pkgs/development/tools/taplo/cli.nix

```diff
@@ -0,0 +1,24 @@
+{ fetchCrate, lib, openssl, pkg-config, rustPlatform, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "taplo-cli";
+  version = "0.4.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "0hh9l83z7qymakyf7ka756gwxpzirgdhf6kpzh89bcmpdfz70005";
+  };
+
+  cargoSha256 = "0bkpcnbrrfv07czs1gy8r9q1cp6fdfz2vmlfk9lsg3iapvyi5s1c";
+
+  nativeBuildInputs = lib.optional stdenv.isLinux pkg-config;
+
+  buildInputs = lib.optional stdenv.isLinux openssl;
```

Comment:
Do you know if it builds on darwin?

Reply:
It's not tested on darwin, but in one of the dependency only require openssl on linux
https://github.com/seanmonstar/reqwest#requirements

Reply:
if it doesn't build on darwin i'll make a patch

Reply:
darwin now builds after some changes